scripts/silo/decorators/decorate_unit_tests.py

```diff
@@ -23,5 +23,9 @@
     ./scripts/decorators/silo/decorate_unit_tests.py < pytest-collect.txt
 """
 
+_MODEL_MANIFEST_FILE_PATH = "./model-manifest.json"  # os.getenv("SENTRY_MODEL_MANIFEST_FILE_PATH")
```

Comment:
I'm a little confused at the environment variable, it's added to the vscode config, but not used anywhere.